[2022-03-18 22:02:52,095] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-03-18 22:02:52,101] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-03-18 22:02:52,101] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 22:02:52,101] {taskinstance.py:1244} INFO - Starting attempt 4 of 5
[2022-03-18 22:02:52,101] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 22:02:52,108] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): extract_task> on 2019-03-01 00:00:00+00:00
[2022-03-18 22:02:52,111] {standard_task_runner.py:52} INFO - Started process 396 to run task
[2022-03-18 22:02:52,113] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EL_dag', 'extract_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/extract_load_dags.py', '--cfg-path', '/tmp/tmp6x42stbg', '--error-file', '/tmp/tmp1d5d9gv8']
[2022-03-18 22:02:52,114] {standard_task_runner.py:77} INFO - Job 45: Subtask extract_task
[2022-03-18 22:02:52,163] {logging_mixin.py:109} INFO - Running <TaskInstance: EL_dag.extract_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host d6b2ec1ac43c
[2022-03-18 22:02:52,191] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.9/site-packages/airflow/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-18 22:02:52,215] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EL_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-03-18 22:02:54,503] {logging_mixin.py:109} INFO - the response is  200
[2022-03-18 22:02:54,503] {logging_mixin.py:109} INFO - here.........
[2022-03-18 22:02:54,538] {logging_mixin.py:109} INFO -    invoice_line_no                     date  ... sale_gallons         store_location
0  INV-17852400013  2019-03-01T00:00:00.000  ...         2.37                   None
1  INV-17861100021  2019-03-01T00:00:00.000  ...         2.77                   None
2  INV-17854400034  2019-03-01T00:00:00.000  ...         0.29                   None
3  INV-17867600038  2019-03-01T00:00:00.000  ...         0.39                   None
4  INV-17858200001  2019-03-01T00:00:00.000  ...         2.37  -95.321174,-95.321174

[5 rows x 24 columns]
[2022-03-18 22:02:54,554] {logging_mixin.py:109} INFO - Data saved..........
[2022-03-18 22:02:54,555] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 22:02:54,566] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=EL_dag, task_id=extract_task, execution_date=20190301T000000, start_date=20220318T220252, end_date=20220318T220254
[2022-03-18 22:02:54,581] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 22:02:54,609] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
